ambry-rest/src/main/java/com.github.ambry.rest/NettyRequest.java

```diff
@@ -67,13 +67,13 @@
   protected final HttpRequest request;
   protected final Channel channel;
   protected final NettyMetrics nettyMetrics;
-  protected final Map<String, Object> allArgs = new TreeMap<String, Object>(String.CASE_INSENSITIVE_ORDER);
-  protected final Queue<HttpContent> requestContents = new LinkedBlockingQueue<HttpContent>();
+  protected final Map<String, Object> allArgs = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```

Comment:
Just chat with @vgkholla , this is to use the feature of `TreeMap` to make keys case insensitive. The order does not matter.